config/database.yml

```diff
@@ -22,7 +22,7 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
```

Comment:
As we don't use multithread mode, only one thread is used so `RAILS_MAX_THREADS ` should be set to 1

Reply:
Thanks @armandfardeau! It's now set to 1